[Build] Fortran module is not installed

**Which version do you want to build?**
Current master branch.

**Which architecture do you want to build for?**
- [x] x86_64

**To Reproduce**
- Changes to `config.mk`: `FORTRAN_INTERFACE = true`
- Changes to build configurations `make/include_GCC.mk`: `FC=gfortran`, uncomment FCFLAGS line for gfortran

**Output log**

`make distclean && make` runs normally. However when trying to run `make install` I see the following:

```
===> INSTALL fortran interface to /home/ivan/.local/include/
cp: cannot stat 'likwid.mod': No such file or directory
```
despite, the `.mod` file being generated:
```
~/fortran/likwid_test/likwid$ ll *.mod
-rw-rw-r-- 1 ivan ivan 787 Okt 22 12:10 likwid.mod
```

When I look at the install PREFIX, the Fortran module has not been installed:
```
~/fortran/likwid_test/likwid$ ll /home/ivan/.local/include/
total 136
drwxr-xr-x 3 ivan ivan   4096 Okt 22 12:13 ./
drwxrwxr-x 9 ivan ivan   4096 Okt 22 12:01 ../
drwxrwxr-x 2 ivan ivan   4096 Mär  8  2023 blis/
-rw-r--r-- 1 root root  14420 Okt 22 12:13 bstrlib.h
-rw-r--r-- 1 root root 101920 Okt 22 12:13 likwid.h
-rw-r--r-- 1 root root   6648 Okt 22 12:13 likwid-marker.h
```

I attempted to just copy the module file to my desired prefix:
```
$ cp likwid.mod /home/ivan/.local/include/
```

If I try linking an application I have symbols missing:
```
~/fortran/likwid_test/likwid$ gfortran -I/home/ivan/.local/include -O3 -o chaos test/chaos.F90 -L/home/ivan/.local/lib -llikwid -lpthread -lm -DLIKWID_PERFMON
/usr/bin/ld: /tmp/ccnTJvdB.o: in function `MAIN__':
chaos.F90:(.text+0xdc): undefined reference to `likwid_markerstart_'
/usr/bin/ld: chaos.F90:(.text+0xed): undefined reference to `likwid_markerstop_'
/usr/bin/ld: chaos.F90:(.text+0xfe): undefined reference to `likwid_markerstart_'
/usr/bin/ld: chaos.F90:(.text+0x15b): undefined reference to `likwid_markerstop_'
/usr/bin/ld: chaos.F90:(.text+0x16c): undefined reference to `likwid_markerstart_'
/usr/bin/ld: chaos.F90:(.text+0x1f9): undefined reference to `likwid_markerstop_'
/usr/bin/ld: chaos.F90:(.text+0x20a): undefined reference to `likwid_markerstart_'
/usr/bin/ld: chaos.F90:(.text+0x266): undefined reference to `likwid_markerstop_'
collect2: error: ld returned 1 exit status
```
Indeed, they appear to be missing from the shared library:

```
~/fortran/likwid_test/likwid$ readelf -Ws --dyn-syms /home/ivan/.local/lib/liblikwid.so | grep likwid_marker
   179: 0000000000225da0  1260 FUNC    GLOBAL DEFAULT   12 likwid_markerInit
   196: 0000000000224f40   871 FUNC    GLOBAL DEFAULT   12 likwid_markerStartRegion
   197: 0000000000232830     9 FUNC    GLOBAL DEFAULT   12 likwid_markerthreadinit_
   213: 0000000000232850     9 FUNC    GLOBAL DEFAULT   12 likwid_markernextgroup_
   216: 0000000000232820     9 FUNC    GLOBAL DEFAULT   12 likwid_markerinit_
   224: 0000000000232920    96 FUNC    GLOBAL DEFAULT   12 likwid_markerstopregion_
   237: 0000000000224d30   325 FUNC    GLOBAL DEFAULT   12 likwid_markerRegisterRegion
   241: 0000000000226290   483 FUNC    GLOBAL DEFAULT   12 likwid_markerThreadInit
   257: 0000000000225980   378 FUNC    GLOBAL DEFAULT   12 likwid_markerGetRegion
   278: 0000000000224480   176 FUNC    GLOBAL DEFAULT   12 likwid_markerNextGroup
   280: 0000000000232840     9 FUNC    GLOBAL DEFAULT   12 likwid_markerclose_
   284: 0000000000232a20    96 FUNC    GLOBAL DEFAULT   12 likwid_markerresetregion_
   287: 0000000000232860    96 FUNC    GLOBAL DEFAULT   12 likwid_markerregisterregion_
   291: 0000000000224530  2044 FUNC    GLOBAL DEFAULT   12 likwid_markerClose
   297: 00000000002328c0    96 FUNC    GLOBAL DEFAULT   12 likwid_markerstartregion_
   312: 00000000002252b0  1729 FUNC    GLOBAL DEFAULT   12 likwid_markerStopRegion
   315: 0000000000232980   150 FUNC    GLOBAL DEFAULT   12 likwid_markergetregion_
   345: 0000000000225b00   443 FUNC    GLOBAL DEFAULT   12 likwid_markerResetRegion
   690: 000000000020fec0    93 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionMetric
   691: 000000000020ff20    93 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionResult
   692: 000000000020ff80    77 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionCount
   693: 000000000020ffd0    74 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionTime
   696: 0000000000210450   350 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionCpulist
   697: 00000000002105b0    46 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionThreads
   698: 00000000002105e0    46 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionEvents
   699: 0000000000210610    46 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionTag
   700: 0000000000210640    49 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerRegionGroup
   701: 0000000000210680    33 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerNumRegions
   702: 00000000002106b0    22 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerFile_destroy
   703: 00000000002106d0    46 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerFile_read
   713: 0000000000210a60    20 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerClose
   714: 0000000000210a80    20 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerNext
   715: 0000000000210aa0    20 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerThreadInit
   716: 0000000000210ac0    20 FUNC    LOCAL  DEFAULT   12 lua_likwid_markerInit
  1797: 0000000000232920    96 FUNC    GLOBAL DEFAULT   12 likwid_markerstopregion_
  1819: 0000000000224d30   325 FUNC    GLOBAL DEFAULT   12 likwid_markerRegisterRegion
  1843: 0000000000232820     9 FUNC    GLOBAL DEFAULT   12 likwid_markerinit_
  1854: 0000000000224480   176 FUNC    GLOBAL DEFAULT   12 likwid_markerNextGroup
  1893: 0000000000232980   150 FUNC    GLOBAL DEFAULT   12 likwid_markergetregion_
  1901: 0000000000224530  2044 FUNC    GLOBAL DEFAULT   12 likwid_markerClose
  1923: 00000000002252b0  1729 FUNC    GLOBAL DEFAULT   12 likwid_markerStopRegion
  1931: 0000000000232a20    96 FUNC    GLOBAL DEFAULT   12 likwid_markerresetregion_
  1935: 0000000000225da0  1260 FUNC    GLOBAL DEFAULT   12 likwid_markerInit
  1951: 0000000000232840     9 FUNC    GLOBAL DEFAULT   12 likwid_markerclose_
  1959: 0000000000224f40   871 FUNC    GLOBAL DEFAULT   12 likwid_markerStartRegion
  1971: 00000000002328c0    96 FUNC    GLOBAL DEFAULT   12 likwid_markerstartregion_
  1981: 0000000000226290   483 FUNC    GLOBAL DEFAULT   12 likwid_markerThreadInit
  1997: 0000000000232830     9 FUNC    GLOBAL DEFAULT   12 likwid_markerthreadinit_
  2005: 0000000000232860    96 FUNC    GLOBAL DEFAULT   12 likwid_markerregisterregion_
  2016: 0000000000225b00   443 FUNC    GLOBAL DEFAULT   12 likwid_markerResetRegion
  2063: 0000000000225980   378 FUNC    GLOBAL DEFAULT   12 likwid_markerGetRegion
  2120: 0000000000232850     9 FUNC    GLOBAL DEFAULT   12 likwid_markernextgroup_
```

--- a/make/config_checks.mk
+++ b/make/config_checks.mk
@@ -63,7 +63,7 @@ FORTRAN_REMOVE_MOVED =
 else
 FORTRAN_IF := $(strip $(FORTRAN_IF_NAME))
 FORTRAN_INSTALL = @echo "===> INSTALL fortran interface to $(PREFIX)/include/"; \
-                  cp -f likwid.mod  $(strip $(PREFIX))/include/$(strip $(FORTRAN_IF_NAME))
+                  cp -f $(BASE_DIR)/likwid.mod  $(strip $(PREFIX))/include/$(strip $(FORTRAN_IF_NAME))
 FORTRAN_REMOVE = @echo "===> REMOVING fortran interface from $(PREFIX)/include/"; \
                  rm -f $(strip $(PREFIX))/include/$(strip $(FORTRAN_IF_NAME))
 FORTRAN_REMOVE_MOVED = @echo "===> REMOVING fortran interface from $(INSTALLED_PREFIX)/include/"; \

